client/components/App.js

```diff
@@ -4,26 +4,32 @@ import { ConnectedRouter } from 'connected-react-router';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import { ThemeProvider } from '@material-ui/core/styles';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import MomentUtils from '@date-io/moment';
 import mainTheme from '../theme';
 import LandingPage from './LandingPage';
 import GamePage from './GamePage';
 import UserNameDialog from './UserNameDialog';
 import WordSelectionDialog from './WordSelectionDialog';
+import StartGameDialog from './StartGameDialog';
 import Snackbars from './Snackbars';
 
 export default ({ history }) => {
   const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
 
   return <ConnectedRouter history={history}>
     <ThemeProvider theme={mainTheme(prefersDarkMode)}>
-      <CssBaseline />
-      <Switch>
-        <Route path='/' exact={true} component={LandingPage} />
-        <Route path='/:id' component={GamePage} />
-      </Switch>
-      <UserNameDialog />
-      <WordSelectionDialog />
-      <Snackbars />
+      <MuiPickersUtilsProvider utils={MomentUtils}>
```

Comment:
what's this part about? Where is MomentUtils used?

Reply:
It's used by the time picker component. It's part of the [getting started](https://material-ui-pickers.dev/getting-started/usage) steps.